hail/python/hail/backend/service_backend.py

```diff
@@ -329,6 +340,9 @@ async def _rpc(self,
                         await write_str(infile, orjson.dumps(reference_config).decode('utf-8'))
                     await write_str(infile, str(self.worker_cores))
                     await write_str(infile, str(self.worker_memory))
+                    await write_int(infile, len(self.regions))
```

Comment:
This breaks if `self.regions` is None, so would need to change if we allow this to be optional